scripts/package-build/blackbox_exporter/package.toml

```diff
@@ -0,0 +1,23 @@
+[[packages]]
+name = "blackbox_exporter"
+commit_id = "v0.25.0"
+scm_url = "https://github.com/prometheus/blackbox_exporter"
+
+build_cmd = """
+
+# Create the install directory
+mkdir -p \
+    debian/usr/sbin \
+    debian/etc/blackbox_exporter
+go build
+
+# Move the blackbox_exporter binary and config to the install directory
+mv blackbox_exporter debian/usr/sbin
+
+# Build the Debian package
```

Comment:
We try to not add more `fpm` related builds.

Could you add a native debian build script isntead?

Reply:
I just followed node_exporter/frr_exporter. Rewrote blackbox_exporter to native debian script.

Reply:
Much appreciated!